ground/gcs/src/plugins/config/configoutputwidget.cpp

```diff
@@ -483,6 +484,7 @@ void ConfigOutputWidget::refreshWidgetsValues(UAVObject *obj)
     }
 
     m_config->motorCurveFit->setValue(actuatorSettingsData.MotorInputOutputCurveFit);
+    m_config->motorPowerGain->setValue(actuatorSettingsData.MotorInputOutputGain * 100);
```

Comment:
This form is still not using uavo relations is it? :(

Reply:
Nope.  And I was scared to do it partially here (is that OK to do, in general?)  Not to mention the form is generated at runtime so that it can't fully be done with auto uavo relations...